ompi/patterns/net/netpatterns.h

```diff
@@ -20,21 +21,21 @@
 
 BEGIN_C_DECLS
 
-int netpatterns_base_err(const char* fmt, ...);
-int netpatterns_register_mca_params(void);
+int ompi_netpatterns_base_err(const char* fmt, ...);
+int ompi_netpatterns_register_mca_params(void);
 
 #if OPAL_ENABLE_DEBUG
-extern int netpatterns_base_verbose; /* disabled by default */
-OMPI_DECLSPEC extern int netpatterns_base_err(const char*, ...) __opal_attribute_format__(__printf__, 1, 2);
+extern int ompi_netpatterns_base_verbose; /* disabled by default */
+OMPI_DECLSPEC extern int ompi_netpatterns_base_err(const char*, ...) __opal_attribute_format__(__printf__, 1, 2);
 #define NETPATTERNS_VERBOSE(args)                                \
     do {                                                         \
-        if(netpatterns_base_verbose > 0) {           \
-            netpatterns_base_err("[%s]%s[%s:%d:%s] ",\
+        if(ompi_netpatterns_base_verbose > 0) {           \
+            ompi_netpatterns_base_err("[%s]%s[%s:%d:%s] ",\
                     ompi_process_info.nodename,                  \
                     OMPI_NAME_PRINT(OMPI_PROC_MY_NAME),          \
                     __FILE__, __LINE__, __func__);               \
             netpatterns_base_err args;               \
```

Comment:
I can see how I missed that in the macro, but what's weird is it didn't show up as "U netpatterns_base_err" in the nm output. Anyway I've changed it.